deployments/docker/src/keycloak.conf

```diff
@@ -0,0 +1,45 @@
+# Basic settings for running in production. Change accordingly before deploying the server.
```

Comment:
May be worth adding a statement as to why these configurations are commented and not used in the conf file.

Reply:
They're are used -- they're commented because the specified values are the defaults, i.e. they are used without needing to be set explicitly. Having them commented shows what the default is, and makes it easy to change the setting. This is a well-known convention for Unix conf files.

But actually, this is a stray file I forgot to delete, as we're using the `keycloak.conf.template` for the deployment, not this. Will remove.